content/en/continuous_testing/troubleshooting.md

```diff
@@ -16,11 +16,14 @@ This page provides information to help you troubleshoot issues with Continuous T
 
 ## Terminology
 
-CI batch
+CI batch 
 : The group of Continuous Testing tests triggered through a continuous integration or continuous delivery (CI/CD) pipeline or the [Datadog Synthetic Monitoring API][2].
 
 Test run
-: A single run of a Continuous Testing test. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.
+: A single run of a Continuous Testing test. This can be [API][7] or [browser tests][8]. If you have configured retries, they count as individual test runs. For example, a test with two retries can have up to three associated test runs.
+
+Parallel test
+: A test that is run at the same time as another test. Parallel tests are part of the Continuous Testing product. In order to control how many tests you would like to run in parallel, configure parallelization in the [settings page][9].
```

Comment:
Trying to rephrase "Parallel tests are part of the Continuous Testing product" in the first sentence.

```suggestion
: A Continuous Testing test that is run at the same time as another Continuous Testing test. To set how many tests you would like to run in parallel, configure parallelization on the [Continuous Testing Settings page][9].
```